$NetBSD: patch-ay,v 1.6 2006/11/14 13:16:00 wennmach Exp $

Change method to determine endianness and adapt to pkgsrc directory structure.

--- src/dbase/grdraster.c.orig	2006-11-13 15:56:13.000000000 +0100
+++ src/dbase/grdraster.c	2006-11-13 15:59:44.000000000 +0100
@@ -50,7 +50,7 @@
 {
 	BOOLEAN xyz_out = FALSE;
 
-#if WORDS_BIGENDIAN == 0
+#if BYTE_ORDER == LITTLE_ENDIAN
 	char my_endian = 'L';	/* This machine is Little endian */
 #else
 	char my_endian = 'B';	/* This machine is Big endian */
@@ -646,7 +646,7 @@
 
 	/* Find and open the file grdraster.info */
 
-	if (!(GMT_getdatapath("grdraster.info", dir) || GMT_getsharepath("dbase", "grdraster", ".info", dir))) {
+	if (!(GMT_getdatapath("grdraster.info", dir) || GMT_getsharepath("gmt/dbase", "grdraster", ".info", dir))) {
 		fprintf(stderr, "%s: ERROR cannot find file grdraster.info", GMT_program);
 		return (0);
 	}
